sdk/communication/communication-phone-numbers/review/communication-phone-numbers.api.md

```diff
@@ -133,6 +142,38 @@ export interface SearchAvailablePhoneNumbersRequest extends PhoneNumberSearchReq
     countryCode: string;
 }
 
+// @public
+export class SipRoutingClient {
+    constructor(connectionString: string, options?: SipRoutingClientOptions);
+    constructor(endpoint: string, credential: KeyCredential, options?: SipRoutingClientOptions);
+    constructor(endpoint: string, credential: TokenCredential, options?: SipRoutingClientOptions);
+    deleteTrunk(fqdn: string, options?: OperationOptions): Promise<void>;
+    getRoutes(options?: OperationOptions): Promise<SipTrunkRoute[]>;
+    getTrunk(fqdn: string, options?: OperationOptions): Promise<SipTrunk | null>;
```

Comment:
What is the scenario for a null response? Are we handling 404s?

Reply:
Can we throw on 404 and let users handle it instead of the client doing so?

Reply:
Null was a solution for getting a not existing trunk. 404 is thrown instead now.